.github/workflows/locker.yml

```diff
@@ -0,0 +1,36 @@
+name: Locker - Lock stale issues and PRs
+on:
+  schedule:
+    - cron: '0 9 * * *' # Once per day, early morning PT
+
+  workflow_dispatch:
+    # Manual triggering through the GitHub UI, API, or CLI
+    inputs:
+      daysSinceClose:
+        required: true
+        default: "30"
+      daysSinceUpdate:
+        required: true
+        default: "30"
+
+permissions:
+  issues: write
+  pull-requests: write
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Actions
+        uses: actions/checkout@v4
+        with:
+          repository: "microsoft/vscode-github-triage-actions"
+          path: ./actions
+          ref: cd16cd2aad6ba2da74bb6c6f7293adddd579a90e
```

Comment:
what's this hash and who updates it?

Reply:
Seems like this refers to a commit here: https://github.com/microsoft/vscode-github-triage-actions/commit/cd16cd2aad6ba2da74bb6c6f7293adddd579a90e

I wonder why we use that instead of just tip of the `stable` branch?

Reply:
Good question. Yes, it's the latest commit sha that affected the locker action from the repo. General guidance from GitHub, the VS Code team, and our own infrastructure folks are all aligned that when reusing actions from another repository (that you don't own), you should stick to a specific commit sha rather than a tag/branch. That ensures your repo always references a known state of the action.

* [Security hardening for GitHub Actions - GitHub Docs](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions):
  * "Pin actions to a full length commit SHA"
* https://github.com/microsoft/vscode-github-triage-actions/?tab=readme-ov-file#vs-codes-issue-triage-github-actions
  * "use the lastest released tag to ensure stability"
  * Except the repo stopped producing new release tags, and release tags don't follow the guidance above since tags can change their commit reference

Reply:
And as far as who updates it: We would only want/need to update it if the locker action is updated to fix a bug or add a feature that we need. Otherwise, it should stay on this commit sha indefinitely.

Reply:
My expectation would have been that we take latest, and then there is some bot (like dependabot) that would submit a PR to update it whenever latest changes.  Can you file an issue in arcade or core-eng requesting that we have some sort of system for keeping these actions up to date?  
Right now this feels like a checked in hardcoded version.